terraform/openstack/hosts/main.tf

```diff
@@ -8,10 +8,11 @@ variable net_id { }
 variable keypair_name { }
 variable image_name { }
 variable control_count {}
-variable resource_count {}
+variable worker_count {}
```

Comment:
Need to refactor L51 as well

    openstack_compute_instance_v2.worker: missing dependency: var.resource_count